docs/code_ref/coordinates/index.rst

```diff
@@ -216,7 +216,7 @@ It is also possible to transform to any coordinate system implemented in Astropy
     >>> greenbelt = EarthLocation(lat=39.0044*u.deg, lon=-76.8758*u.deg)
     >>> greenbelt_frame = AltAz(obstime=time, location=greenbelt)
     >>> west_limb = SkyCoord(900*u.arcsec, 0*u.arcsec, frame=frames.Helioprojective, obstime=time)
-    >>> west_limb.transform_to(greenbelt_frame)  # doctest: +SKIP
+    >>> west_limb.transform_to(greenbelt_frame)
```

Comment:
This breaks:
```
UNEXPECTED EXCEPTION: ConvertError('This transformation cannot be performed because the Helioprojective frame has observer=None.',)
```

Reply:
Weird.  Why was this ever skipped instead of done correctly?  The line above it should be:
```python
>>> west_limb = SkyCoord(900*u.arcsec, 0*u.arcsec, frame=frames.Helioprojective,
...                      observer=greenbelt.get_itrs(greenbelt_frame.obstime), obstime=time)
```

Reply:
Ah, because it triggered a download of the IERS file, and the message broke the doctest (see https://github.com/sunpy/sunpy/commit/74f46996b464c69144826dea457736f46ea887cb).  If this is still a problem, we just need to manually pre-download the file in a doctest setup.

Reply:
We seem to skip a bunch of things in our docs (something I also plan to address one day).

Thanks for the fix, I will apply it.

Reply:
Ah I remember, I will try to work around locally.

Reply:
The online build seems to be ok with the example.